src/core/scene/a-scene.js

```diff
@@ -674,6 +673,11 @@ module.exports.AScene = registerElement('a-scene', {
           };
         }
 
+        // Even if the user wants webgl2, if it's not available then fall back to webgl1.
+        if (!document.createElement('canvas').getContext('webgl2')) {
```

Comment:
Potentially if we want to avoid doing needless work, we could say `if (useWebGL2 && !document...)` so if it's already false we don't need to make a new context.

Reply:
Updated.